pkg/generated/openapi/def.bzl

```diff
@@ -19,26 +19,27 @@ def openapi_library(name, tags, srcs, go_prefix, vendor_prefix = "", openapi_tar
     deps = [
         "//vendor/github.com/go-openapi/spec:go_default_library",
         "//vendor/k8s.io/kube-openapi/pkg/common:go_default_library",
-    ] + ["//%s:go_default_library" % target for target in openapi_targets] + ["//vendor/%s:go_default_library" % target for target in vendor_targets]
+    ] + ["//%s:go_default_library" % target for target in openapi_targets] + ["//staging/src/%s:go_default_library" % target for target in vendor_targets]
     go_library(
         name = name,
         srcs = srcs + [":zz_generated.openapi"],
+        importpath = go_prefix + "pkg/generated/openapi",
         tags = tags,
         deps = deps,
     )
     go_genrule(
         name = "zz_generated.openapi",
-        srcs = srcs + ["//" + vendor_prefix + "hack/boilerplate:boilerplate.go.txt"],
```

Comment:
what's with dropping `srcs` here? did we ever need these for this rule?

Reply:
I don't think we need them for the generator (currently it's only `doc.go`) - they're only needed for the `go_library` itself.